server/schedule/filter/filters.go

```diff
@@ -601,3 +601,74 @@ const (
 
 var allSpecialUses = []string{SpecialUseHotRegion, SpecialUseReserved}
 var allSpeicalEngines = []string{EngineTiFlash}
+
+type isolationFilter struct {
+	scope         string
+	constraintSet [][]*placement.LabelConstraint
```

Comment:
I think no need to introduce `placement.LabelConstraint` here. We can just use `[][]string`.